build/yamls/flow-aggregator/base/conf/flow-aggregator.conf

```diff
@@ -34,3 +34,8 @@
 # Provide format for records sent to the configured flow collector.
 # Supported formats are IPFIX and JSON.
 #recordFormat: "IPFIX"
+
+# recordContents contains flow records related configuration options.
```

Comment:
"recordContents enables configuring some fields in the flow records, A field can be either included (true) or excluded (false) by providing a bool value."

Reply:
done